uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -818,6 +883,10 @@ define([
             breakdown: []
           }
         };
+
+        if (vm.isleaveType('sick')) {
+          vm.leaveRequest.reason = null;
```

Comment:
1. all selections are reset when user changes from multiple to single and vice versa
2. document is not edited by owner